stack/dashboard/base/files/etc/plugins_install_settings.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
```

Comment:
Please take into consideration to make `sed` instead of having the entire file because be aware and import automatically changes on this file when building a new base-based future Opensearch versions.

Reply:
Done: 5edb32d23ad5c015d5d9f257a4e32e4131250714